go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	go.etcd.io/etcd/client/v3 v3.5.1
 	google.golang.org/grpc v1.43.0
 	k8s.io/api v0.23.5
-	k8s.io/apimachinery v0.23.5
+	k8s.io/apimachinery v0.24.0
```

Comment:
Why this upgrade?

Reply:
Don't know. Maybe because I was using go1.17 and then I switched to go1.18.
I'm going to restore it with the previous version.